src/isIsogram.test.js

```diff
@@ -8,12 +8,26 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    expect(typeof isIsogram('')).toBe('boolean');
+  });
 
+  it(`should not be case sensitive`, () => {
+    expect(isIsogram('DOGE')).toBe(true);
```

Comment:
```suggestion
    expect(isIsogram('DdOGE')).toBe(true);
```
отак буде правильно перевірити сенситивність